CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Update favicon to be the Elastic Package Registry logo. [#858](https://github.com/elastic/package-registry/pull/858)
 * Implement proxy mode. [#860](https://github.com/elastic/package-registry/pull/860)
+* Proxy mode doesn't need local packages. [#871](https://github.com/elastic/package-registry/pull/871)
```

Comment:
I mean, not needed at all :smile: I think that regarding user-facing changes this is covered by the "Implement proxy mode" line.

If you want to include the changeset I guess you can add it to the previous line.
```suggestion
* Implement proxy mode. [#860](https://github.com/elastic/package-registry/pull/860) [#871](https://github.com/elastic/package-registry/pull/871)
```